docs/ReleaseHistory.md

```diff
@@ -16,7 +16,15 @@
 - BRK, NEW: `SecretMasker.MaskSecrets` and `ISecretMasker.DetectSecrets` now accept an `Action<Detection>` callback (with a default value of `null`) to receive detections that result from the operation. This is binary-breaking only for `SecretMasker` callers and source-breaking as well for `ISecretMasker` implementations.
 - BRK: `Detection` is now sealed and immutable. All property setters and the copy constructor are removed.
 - BRK: `Detection`  no longer overrides `Equals` and `GetHashCode` nor implements `IEquatable<Detection>`. `Detection.Equals` and `Detection.GetHashCode` will therefore now be based on reference equality.
+- BRK, PRF: `IdentifiableScan` has been removed and it's functionality has been merged into `SecretMasker` which now scans the patterns it's given as fast as possible, which will be significantly faster for identifiable key patterns.
```

Comment:
>it's [](http://example.com/codeflow?start=52&length=4)

This s/be `its`, best wishes, the `it's vs its` police. Sorry I missed this class D misdemeanor earlier